compose/cli/main.py

```diff
@@ -137,14 +151,13 @@ def setup_logging():
     logging.getLogger("requests").propagate = False
 
 
-def setup_parallel_logger(noansi):
-    if noansi:
-        import compose.parallel
-        compose.parallel.ParallelStreamWriter.set_noansi()
+def setup_parallel_logger(ansi):
+    import compose.parallel
+    compose.parallel.ParallelStreamWriter.set_ansi(ansi)
 
 
-def setup_console_handler(handler, verbose, noansi=False, level=None):
-    if handler.stream.isatty() and noansi is False:
+def setup_console_handler(handler, verbose, ansi=AnsiMode.auto, level=None):
+    if ansi == AnsiMode.always or (ansi == AnsiMode.auto and handler.stream.isatty()):
```

Comment:
```suggestion
    if ansi.use_tty(handler.stream):
```

Or other method name defined in the enum